qtcolorpicker/src/qtcolorpicker.cpp

```diff
@@ -309,8 +309,8 @@ void QtColorPicker::buttonPressed(bool toggled)
 {
     if (!toggled)
         return;
-
-    const QRect desktop = QApplication::desktop()->geometry();
+    const QScreen* qscr = QApplication::primaryScreen();
```

Comment:
It may be better if this used [screenAt](https://doc.qt.io/qt-5/qguiapplication.html#screenAt) instead of primaryScreen.